.github/workflows/reminders.yml

```diff
@@ -13,6 +13,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          webhook-url: ${{ secrets.SLACK_WEBHOOK }}
-          provider: slack
-          github-provider-map: 'fw-noel:UUWMCRU4Q,taraepp:U0373NCPMPH,arranfw:U01ADH9FUET,alva-fresh:U012X7BJV0T'
+          webhook-url: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
+          provider: msteams
+          github-provider-map: 'fw-noel:Noel Feliciano,taraepp:Tara Epp,arranfw:Arran Woodruff,alva-fresh:Alva Kostrova'
```

Comment:
I have them all being sent to the same channel, yes. Can make a 2nd secret if we want them sent to multiple channels, not a big deal (but I do need someone with more GH permissions than me to create it in GH). 

As far as the mention support... I had not seen before any indication that adaptive card format could be used with incoming webhooks, so that's interesting! Perhaps "explore all possible messaging venues with GH & Teams" would make for a more elaborate hackathon project than I originally thought.